sqlite3_cache/cache.py

```diff
@@ -522,3 +525,99 @@ def ttl_many(self, keys: List[str]) -> Dict[str, int]:
             self._con.commit()
 
         return results
+
+    def _filter_key_result_list(self, unfiltered: List[Tuple[str, Any]]) -> List[str]:
+        """
+        Filters key result list to only those keys for cache items that are still alive,
+        and purges expired items from the cache
+        :param unfiltered: A list of key and expiration tuples
```

Comment:
Space before `:params` (should be a ruff lint rule, don't know why it didn't catch this).

Reply:
Fixed

Reply:
Sorry, I wasn't precise enough: There _**should**_ be a newline between the body and the params. Could you add them back?

Reply:
and now fixed the proper way to add space where it was missing